General styling tweaks across the site

BenPoole88 opened the pull request:
- Changed banner image to strip_workers
- Added margin to bottom of Division image block
- Reduced "p" size to 16px across site
- Removed top/bottom padding from footer
- Changed footer font weight to 400

![Screenshot 2021-03-09 at 14 27 52 (2)](https://user-images.githubusercontent.com/58986635/110488672-95dc2c80-80e6-11eb-9300-6678375caa69.png)

- Changed nav button font weight to 400

![Screenshot 2021-03-09 at 14 34 16 (2)](https://user-images.githubusercontent.com/58986635/110488743-a9879300-80e6-11eb-91f3-774c9e03da7f.png)

- Applied job teaser styling to full job page

![Screenshot 2021-03-09 at 14 37 10 (2)](https://user-images.githubusercontent.com/58986635/110488805-befcbd00-80e6-11eb-83ee-cf1e5f53aa2c.png)

BenPoole88 committed:
by Ben Poole: exposed Keyword and Function filters on Job page

BenPoole88 committed:
by Ben Poole: styling exposed filters on jobs page

BenPoole88 committed:
by Ben Poole: general styling tweaks across the site

BenPoole88 committed:
by Ben Poole: added strong tags to Divisions text

BenPoole88 commented:
Added strong tags to Divisions page. A tag added also, but needs a href still.

![Screenshot 2021-03-09 at 16 26 52 (2)](https://user-images.githubusercontent.com/58986635/110510743-6a177180-80fb-11eb-9f94-a3331d9eaad8.png)

BenPoole88 committed:
by Ben Poole: further styling tweaks to jobs page

BenPoole88 commented:
- Removed strong tag from header text and changed size/weight
- Renamed 'Apply' to 'Search Jobs' and exposed Reset button

![Screenshot 2021-03-10 at 10 36 22 (2)](https://user-images.githubusercontent.com/58986635/110617101-3553fa80-818d-11eb-939c-90473e8fbc03.png)

BenPoole88 committed:
by Ben Poole: further styling tweaks to site

BenPoole88 commented:
- Changed footer font size to initially to be 10px rather than 14px, to accommodate for mobile up to larger breakpoint

![Screenshot 2021-03-10 at 10 48 04 (2)](https://user-images.githubusercontent.com/58986635/110624700-78ff3200-8196-11eb-8ea4-c82216b88d34.png)

- Added a fixed width of 130px to nav buttons so they're uniform

![Screenshot 2021-03-10 at 11 01 17 (2)](https://user-images.githubusercontent.com/58986635/110624776-8d432f00-8196-11eb-905b-d1e8307f8fe6.png)

- Removed margins from p as there was an odd margin in Testimonials

![Screenshot 2021-03-10 at 11 09 27 (2)](https://user-images.githubusercontent.com/58986635/110624854-a8ae3a00-8196-11eb-932e-a4eb004852ed.png)

- Changed banner image on Contact form

![Screenshot 2021-03-10 at 11 17 07 (2)](https://user-images.githubusercontent.com/58986635/110624919-b8c61980-8196-11eb-845d-096d1d7692fe.png)

BenPoole88 committed:
by Ben Poole: changed Divisions to About Us

rlmumford commented on web/modules/custom/oc_pages/oc_pages.links.menu.yml:
```diff
@@ -1,5 +1,5 @@
 oc_pages.divisions:
```
You should rename this to `oc_pages.about`

rlmumford commented on web/modules/custom/oc_pages/oc_pages.routing.yml:
```diff
@@ -6,10 +6,10 @@ oc_pages.home:
   requirements:
     _access: 'TRUE'
 oc_pages.divisions:
```
Rename to `oc_pages.about` - remember to update the links.menu.yml file

rlmumford commented on web/themes/custom/oc_bootstrap/oc_bootstrap.theme:
```diff
@@ -33,7 +33,7 @@ function oc_bootstrap_preprocess_page(&$variables) {
       $variables['banner_image_class'] = 'image-'.$node->redfin_header_image->value;
     }
-    else if ($node->bundle() == 'division') {
+    else if ($node->bundle() == 'about') {
```
Don't need this change, we haven't changed the node type from division to about.

rlmumford commented on web/themes/custom/oc_bootstrap/oc_bootstrap.theme:
```diff
@@ -43,7 +43,7 @@ function oc_bootstrap_preprocess_page(&$variables) {
  */
 function oc_bootstrap_preprocess_menu(&$variables) {
-  if (!isset($variables['menu_name']) || $variables['menu_name'] !== 'divisions') {
+  if (!isset($variables['menu_name']) || $variables['menu_name'] !== 'about') {
```
We haven't change the menu name from divisions to about, so don't need this change.

BenPoole88 committed:
by Ben Poole: further styling updates across site

BenPoole88 commented:
- Added "Contact" button to navbar (which will go to http://olivercarol.com/contact)

![Screenshot 2021-03-10 at 16 48 26 (2)](https://user-images.githubusercontent.com/58986635/110667514-3ce1c680-81c2-11eb-9948-c03a2d28ac41.png)

- Fixed the requested points above, and added banner image back in:

![Screenshot 2021-03-10 at 16 45 51 (2)](https://user-images.githubusercontent.com/58986635/110667059-c93fb980-81c1-11eb-81d6-4f15597f0f18.png)

- Additional styling to job filters. **Need a little help here please to get the two inputs 50% width each for 990px upwards**:

![Screenshot 2021-03-10 at 16 47 02 (2)](https://user-images.githubusercontent.com/58986635/110667250-fbe9b200-81c1-11eb-909b-3f4acabe1486.png)

- Added banner image back in to Testimonials page:

![Screenshot 2021-03-10 at 16 47 28 (2)](https://user-images.githubusercontent.com/58986635/110667365-1459cc80-81c2-11eb-899c-7d015368f5cb.png)

- Updated placeholder text for Phone number in contact form, made all inputs full width, and styled buttons:

![Screenshot 2021-03-10 at 16 45 13 (2)](https://user-images.githubusercontent.com/58986635/110667656-60a50c80-81c2-11eb-8ef4-c670df139486.png)

BenPoole88 committed:
by Ben Poole: updated truck.png

BenPoole88 committed:
by Ben Poole: fixed minor image breakpoint issues

BenPoole88 commented:
I've pushed a few more tweaks this morning, and in doing so somehow I've lost the job teaser beneath the filters, I can't figure out how to amend this in the view though. Can you help please?

![Screenshot 2021-03-11 at 09 32 09 (2)](https://user-images.githubusercontent.com/58986635/110766491-22086400-824d-11eb-94b2-f8e95b5b1a6f.png)

BenPoole88 committed:
by Ben Poole: fixed background issue on Testimonials page

rlmumford commented on web/modules/custom/oc_pages/src/Controller/PagesController.php:
```diff
@@ -41,13 +41,13 @@ public function divisions() {
         '#tag' => 'p',
         '#value' => new TranslatableMarkup('If you are a business looking to add talent to your team and want a
-        knowledgeable, dedicated, and ethical recruitment partner contact us and let’s explore how we can serve you.'),
+        knowledgeable, dedicated, and ethical recruitment partner <a href="http://www.olivercarol.com/contact"><strong>contact us</strong></a> and let’s explore how we can serve you.'),
```
This should be done by adding a token for the URL.

```php
new TranslatableMarkup(
  'blah blah blah <a href=":url"><strong>link text</strong></a>  blah blah blah',
  [
    ':url' => Url::fromRoute('contact.site_page'),
  ]
);
```

rlmumford replied:
We basically never want the domain to be written in code anywhere

BenPoole88 committed:
by Ben Poole: added /contact URL token to About Us page
